eos-tech-support/eos-diagnostics

```diff
@@ -558,6 +558,11 @@ let diagnostics = [
         hardwareInfo: true,
         content: function() { return trySpawn('journalctl --no-pager --output verbose'); },
     },
+    {
+        title: 'Coredump',
```

Comment:
Modified it to output a warning message and some stub content if `systemd-coredump` is not enabled in the kernel’s `core_pattern` sysctl.